pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix

```diff
@@ -97,17 +97,19 @@ let
         "https://dist.torproject.org/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
         "https://tor.eff.org/dist/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
         "https://tor.calyxinstitute.org/dist/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
+        "https://archive.torproject.org/tor-package-archive/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
```

Comment:
```suggestion
        "https://dist.torproject.org/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
        "https://archive.torproject.org/tor-package-archive/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
        "https://tor.eff.org/dist/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
        "https://tor.calyxinstitute.org/dist/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
```

Reply:
the other mirrors can occasionally be useful if `torproject.org` is blocked (which happens in the wild for some dumb fortinet configs) so imho some redundancy is better

Reply:
Okay but then I would still move archive.torporject.org as an second link incase the first one is removed again.

Reply:
That's fine by me

Reply:
I prefer to keep dist.torproject.org first, because it is the official download server. I could imagine that archive.torproject.org may be a bit less powerful than the former, although I have no proof for that. For example, neither the [readme](https://archive.torproject.org/README.txt) nor the (old) [blog post](https://blog.torproject.org/archivetorprojectorg-now-live/) mention bandwidth limitations. So maybe the purpose of dist.torproject.org vs. archive.torproject.org is just to make mirroring Tor Project's files easier and less expensive for those who want to provide a mirror.